internal/static/integrations/catchpoint_exporter/catchpoint_exporter.go

```diff
@@ -0,0 +1,66 @@
+package catchpoint_exporter
+
+import (
+	"github.com/go-kit/log"
+	"github.com/grafana/alloy/internal/static/integrations"
+	integrations_v2 "github.com/grafana/alloy/internal/static/integrations/v2"
+	"github.com/grafana/alloy/internal/static/integrations/v2/metricsutils"
+
+	collector "github.com/grafana/catchpoint-prometheus-exporter/collector"
+)
+
+// DefaultConfig is the default config for the catchpoint integration
+var DefaultConfig = Config{
+	VerboseLogging: false,
+	WebhookPath:    "/catchpoint-webhook",
+	Port:           "9090",
+}
+
+// Config is the configuration for the catchpoint integration
+type Config struct {
+	VerboseLogging bool   `yaml:"verbose_logging,omitempty"`
+	WebhookPath    string `yaml:"webhook_path,omitempty"`
+	Port           string `yaml:"port,omitempty"`
+}
```

Comment:
just fyi some code from this file is useless like the one above this comment because the yaml part was only relevant for static mode. But since that we did not update the exporters yet to get rid of the legacy part I think it makes sense to do it like this and keep the exporters consistent.

Reply:
So I should keep it as is?

Reply:
yes I think this should be kept as it is